
** (second.py:5284): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-MmuWRdOEEf: Connection refused



  variant12.append( Sample( lambda x: (x[0] - x[1])**2 + ((x[0] + x[1] - 10)**2) / 9.0, [0, 1], [[2 + 2.0/9.0, -2 + 2.0/9.0], [-2 + 2.0/9.0, 2 + 2.0/9.0]] ) )

h = 10^-2 

Traceback (most recent call last):
  File "./second.py", line 110, in <module>
    main()
  File "./second.py", line 99, in main
    variant12[j].drawPlot(2.5, 25)
  File "./second.py", line 34, in drawPlot
    facecolors=cm.jet(N1),
NameError: global name 'cm' is not defined
/usr/local/lib/python2.7/dist-packages/matplotlib/backends/backend_gtk3.py:215: Warning: Source ID 2 was not found when attempting to remove it
  GLib.source_remove(self._idle_event_id)
